.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,76 @@
+name: Process Benchmark
+
+on:
+  workflow_dispatch:
+    inputs:
+      benchmark_data:
+        description: 'Base64 encoded benchmark data'
+        required: true
+      benchmark_name:
+        description: 'Name for benchmark group on the dashboard'
+        required: false
+        default: 'RAJAPerf benchmarks'
+
+permissions:
+  checks: write
+  contents: write
+  pull-requests: write
```

Comment:
Double checking: are all those permissions required?

Reply:
Good point! The `checks: write` allows the adding of a PR check status. I will remove the `pull-requests: write`-- that is only needed for writing comments to commits/PRs. `contents: write` is needed to push to the gh-pages branch. We only want to do that on `develop` but I don't think you can make permissions conditional. So now I split this into two jobs, one for PRs and one for develop-- there will be a bit of duplication but then each can have the correctly scoped permissions. Let me know what you think!